py/client/pydeephaven/session.py

```diff
@@ -360,13 +333,13 @@ def plugin_object_service(self) -> PluginObjService:
                     self._plugin_obj_service = PluginObjService(self)
         return self._plugin_obj_service
 
-    def make_ticket(self, ticket_no=None):
+    def make_export_ticket(self, ticket_no=None) -> ExportTicket:
         if not ticket_no:
-            ticket_no = self.get_ticket()
+            ticket_no = self.next_export_ticket_number()
         ticket_bytes = ticket_no.to_bytes(4, byteorder='little', signed=True)
```

Comment:
Should ownership of the formatting move to `ExportTicket`, like it has for other ticket types?